src/Dfe.PlanTech.Web/Controllers/RecommendationsController.cs

```diff
@@ -0,0 +1,30 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Dfe.PlanTech.Web.Routing;
+
+namespace Dfe.PlanTech.Web.Controllers;
+
+[Authorize]
+public class RecommendationsController : BaseController<RecommendationsController>
+{
+  public const string GetRecommendationAction = nameof(GetRecommendation);
+
+  public RecommendationsController(ILogger<RecommendationsController> logger) : base(logger)
+  {
+  }
+
+  [HttpGet("{sectionSlug}/recommendation/{recommendationSlug}", Name = "GetRecommendation")]
+  public Task<IActionResult> GetRecommendation(string sectionSlug,
+                                                     string recommendationSlug,
+                                                     [FromServices] IGetRecommendationRouter getRecommendationValidator,
+                                                     CancellationToken cancellationToken)
+  {
+    if (string.IsNullOrEmpty(sectionSlug)) throw new ArgumentNullException(nameof(sectionSlug));
+    if (string.IsNullOrEmpty(recommendationSlug)) throw new ArgumentNullException(nameof(recommendationSlug));
```

Comment:
Contentful currently considers one or more spaces to be valid for the Slug field on pages. I dunno if we can change that or if its a limitation, but would `string.IsNullOrWhitespace(..)` possibly be a safer bet here? It includes null, empty or whitespaces in its check rather than just null or empty.

Reply:
Good shout, but thankfully ASP NET Core won't like it regardless.

E.g.
<img width="420" alt="image" src="https://github.com/DFE-Digital/plan-technology-for-your-school/assets/131891854/bc286f4e-52a6-4dee-bf7e-b53e7d118e95">

<img width="372" alt="image" src="https://github.com/DFE-Digital/plan-technology-for-your-school/assets/131891854/d275ddd2-11e3-488d-8cb7-5325f99fc98d">